src/test/playwright/package.json

```diff
@@ -4,15 +4,16 @@
     "private": true,
     "license": "MIT",
     "devDependencies": {
-        "@playwright/test": "1.40.0",
-        "dayjs": "1.11.10",
-        "dotenv": "16.3.1",
-        "simple-git": "3.22.0",
+        "@playwright/test": "1.44.0",
+        "dayjs": "1.11.11",
+        "dotenv": "16.4.5",
+        "simple-git": "3.24.0",
         "uuid": "9.0.1"
     },
     "scripts": {
         "playwright:test": "playwright test e2e",
         "playwright:open": "playwright test e2e --ui",
-        "playwright:setup": "npx playwright install --with-deps chromium && playwright test init"
+        "playwright:setup": "npx playwright install --with-deps chromium && playwright test init",
+        "update": "ncu -i --format group"
```

Comment:
I could't run `npm run update` on both windows and linux. I think it would be best to include `ncu` in dev dependencies or add a comment on how to use this script.

Reply:
Added ncu as a dev dependency 👍